service/frontend/workflow_handler.go

```diff
@@ -3865,6 +3868,7 @@ func (wh *WorkflowHandler) StartBatchOperation(
 		CancelParams:    batcher.CancelParams{},
 		SignalParams:    signalParams,
 		DeleteParams:    batcher.DeleteParams{},
+		ResetParams:     batcher.ResetParams{},
```

Comment:
Need to wire up the fields in the reset parameter.

Reply:
These are all wired up later in the code when we check the request type.